framework/asserts/registration_asserts.py

```diff
@@ -1,19 +1,24 @@
 from hamcrest import assert_that, is_
+import bcrypt
 
 
 def check_mapping_api_to_db(api_request: dict, database_data: dict) -> None:
-    """Checking the mapping of data from the request API to database
+    """Checking the mapping of data from the request API to the database
 
     Args:
-        api_request:    reference data, data from an API-request;
-        database_data:  compared data, data from database.
+        api_request:    reference data, data from an API request;
+        database_data:  compared data, data from the database (a dictionary).
     """
     fields_api_to_db = {
         "email": "email",
         "firstName": "first_name",
         "lastName": "last_name",
         "password": "password",
```

Comment:
Checking the password at 19 line => need delete this line (`"password": "password",`)

Reply:
Row 19 doesn't verify if the passwords match; it just hashes the new user's password using the database password and new user password as the salt. However, password comparison still occurs within a loop.

Reply:
I apologize. I didn't understand the situation ((